### PR TITLE
examples: linux: cmdline support for rpmsg ctrl dev

### PR DESCRIPTION
**Pull request by rajimoha**
Adding option -c for passing the rpmsg control device which the application needs to use.

Signe-off-by: Rajiv Mohan <rajiv.mohan@amd.com>

### CODE REVIEW

**Review by arnopo — APPROVED** [review]
LGTM

**Review by tnmysh — CHANGES_REQUESTED** [review]

**Review by tnmysh — APPROVED** [review]
LGTM.

File: examples/linux/rpmsg-echo-test/echo_test.c [hunk 1]
```diff
@@ -94,11 +95,16 @@ int main(int argc, char *argv[])
...
 
-	while ((opt = getopt(argc, argv, "d:n:s:e:")) != -1) {
+	while ((opt = getopt(argc, argv, "d:c:n:s:e:")) != -1) {
 		switch (opt) {
 		case 'd':
+			memset(rpmsg_dev, 0, sizeof(rpmsg_dev));
 			strncpy(rpmsg_dev, optarg, sizeof(rpmsg_dev));
 			break;
+		case 'c':
+			memset(rpmsg_ctrl_dev_name, 0,sizeof(rpmsg_ctrl_dev_name));
```
Comment (arnopo) [inline examples/linux/rpmsg-echo-test/echo_test.c:105]: a space is missing before sizeof

**Review by arnopo — CHANGES_REQUESTED** [review]
checkpatch reports 2 issues:
sapace missing (see my comment
s/signe-off-by/signed-of-by/ in the commit message 

please fix before I merge it

**Review by arnopo — APPROVED** [review]

**Pull request merged**